src/applications/vaos/actions/newAppointment.js

```diff
@@ -89,6 +83,13 @@ export const START_DIRECT_SCHEDULE_FLOW =
   'newAppointment/START_DIRECT_SCHEDULE_FLOW';
 export const START_REQUEST_APPOINTMENT_FLOW =
   'newAppointment/START_REQUEST_APPOINTMENT_FLOW';
+export const FORM_CALENDAR_FETCH_SLOTS =
+  'newAppointment/FORM_CALENDAR_FETCH_SLOTS';
+export const FORM_CALENDAR_FETCH_SLOTS_SUCCEEDED =
+  'newAppointment/FORM_CALENDAR_FETCH_SLOTS_SUCCEEDED';
+export const FORM_CALENDAR_FETCH_SLOTS_FAILED =
+  'newAppointment/FORM_CALENDAR_FETCH_SLOTS_FAILED';
+export const FORM_CALENDAR_ON_CHANGE = 'newAppointment/FORM_CALENDAR_ON_CHANGE';
```

Comment:
We're not super consistent about this (I'll add a ticket to clean that up), but I think we want this to be past tense. So like `FORM_CALENDAR_DATA_CHANGED`. The action creator function name can stay the way it is, I think.